packages/react-microlink/src/index.js

```diff
@@ -5,23 +5,18 @@ import {CardWrap, CardMedia, CardContent, CardEmptyState} from './components/Car
 import {getUrlPath, someProp, createApiUrl} from './utils'
 
 class Microlink extends Component {
-  constructor (props) {
-    super(props)
-    this.setData = this.setData.bind(this)
-  }
-
   componentWillMount () {
     if (this.props.data) return this.setData({data: this.props.data})
 
     const url = createApiUrl(this.props)
-    this.setState({loading: true})
+    this.setState({loading: true}, this.fetchData)
 
     return fetch(url, {headers: {'x-api-key': this.props.apiKey}})
       .then(res => res.json())
       .then(this.setData)
   }
 
-  setData ({data}) {
+  setData = ({data}) => {
```

Comment:
I meant on line 16, changing the `then()` to `then(data => this.setData(data))`

Reply:
why? I'm passing the `data` directly

Reply:
Yeah yeah, your current solution is fine 👌. Just saying that's what I _originally_ meant when I said about removing the need for `.bind(this)`